bundles/org.openhab.binding.mycroft/README.md

```diff
@@ -0,0 +1,161 @@
+# Mycroft Binding
+
+This binding will connect to Mycroft A.I. in order to control it or react to event by listening on the message bus.
+
+Possibilies include :
+
+- Press a button in OpenHAB to wake Mycroft without using a wake word.
+- Simulate a voice command to launch a skill, as if you just spoke it
+- Send some text that Mycroft will say (Using its Text To Speach service)
+- Control the music player
+- Control the sound volume of Mycroft
+- React to all the aforementioned events ...
+- ... And send/receive all other kind of messages on the message bus
+
+
+## Supported Things
+
+The only thing managed by this binding is a Mycroft instance.
+
+
+## Discovery
+
+There is no discovery service, as Mycroft doesn't announce itself on the network.
+
+
+## Thing Configuration
+
+The configuration is simple, as you just need to give the IP/hostname of the Mycroft instance accessible on the network.
+The default port is 8181, but you could change it if you want.
+
+```
+Thing mycroft:mycroft:myMycroft "Mycroft A.I." @ "Living Room" [host="192.168.X.X"]
+```
+
+|   property    |              type               |         description                                                     | mandatory |
+|---------------|---------------------------------|-------------------------------------------------------------------------|-----------|
+| host          | IP or string                    | IP address or hostname                                                  |   Yes     |
+| port          | integer                         | Port to reach Mycroft (default 8181)                                    |   No      |
+
+
+## Channels
+
+The Mycroft thing supports the following channels :
+
+
+| channel type id              | Item type | description                                                                                    |
+|------------------------------|-----------|------------------------------------------------------------------------------------------------|
+| listen                       | Switch    | Switch to ON when Mycroft is listening. Can simulate a wake word detection to trigger the STT  |
+| speak                        | String    | The last sentence Mycroft speaks, or ask Mycroft to say something                              |
+| utterance                    | String    | The last utterance Mycroft receive, or ask Mycroft something                                   |
+| player                       | Player    | The music player Mycroft is currently controlling                                              |
+| volume                       | Dimmer    | The volume of the Mycroft speaker. NOT FUNCTIONNAL. [SEE THIS POST TO SEE EVOLUTION](https://community.mycroft.ai/t/openhab-plugin-development-audio-volume-message-types-missing/10576) |
```

Comment:
If this is still not working I would leave it out of the README and thing definition for now. You can keep the code, but please hide this functionality from the user until its working.

Reply:
Totally agree, thanks